.env.example

```diff
@@ -8,10 +8,17 @@ NEXT_PUBLIC_IPFS_BASE_URL=https://ipfs.io/ipfs/
 NEXT_PUBLIC_POH_SUBGRAPH_URL=https://api.thegraph.com/subgraphs/name/kleros/proof-of-humanity-mainnet
 NEXT_PUBLIC_SISMO_GRAPH_API=https://api.sismo.io/
 NEXT_PUBLIC_SIGNATURE_API_URL=https://api.defender.openzeppelin.com/autotasks/4b1688f9-01a4-435d-89ae-d05e0aa0a53b/runs/webhook/b9818d77-3c43-4c3d-bfb8-4c77036de92f/ACXhXsQoCKP8zZVE26gFbh
+NEXT_PUBLIC_BACKEND_RPC_URL=https://rpc-mumbai.maticvigil.com/v1/xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
 
 # MESSENGING
 ##Can be 'prod', 'staging' or 'dev' for PUSH || Can be 'production', 'local' or 'dev' for XMTP
 NEXT_PUBLIC_MESSENGING_ENV='dev'
 
+# DELEGATION
+NEXT_PUBLIC_ACTIVE_DELEGATE=true
```

Comment:
here we configure already two things for delegate, it just miss the public address, make more sense to have everything here